src/components/NavBar/MenuItemMobile/index.tsx

```diff
@@ -0,0 +1,184 @@
+            <Typography variant="body2">{title}</Typography>
+          </AccordionSummary>
+          <AccordionDetails className={classes.accordionDetail}>
+            <Grid container direction="column">
+              {layersCategories.map(
+                ({ title: categoryTitle, layers, tables }) => (
+                  <Grid
+                    item
+                    key={categoryTitle}
+                    className={classes.categoryContainer}
+                  >
+                    <Typography
+                      variant="body2"
+                      className={classes.categoryTitle}
+                    >
+                      {categoryTitle}
+                    </Typography>
+                    <hr />
+
+                    {layers.map(layer => {
+                      const { id: layerId, title: layerTitle } = layer;
+                      const selected = Boolean(
+                        selectedLayers.find(
+                          ({ id: testId }) => testId === layerId,
+                        ),
+                      );
+                      return (
+                        <div key={layerId} className={classes.layersContainer}>
+                          <Switch
+                            size="small"
+                            color="default"
+                            checked={selected}
+                            onChange={() => toggleLayerValue(selected, layer)}
+                            inputProps={{ 'aria-label': layerTitle }}
+                          />{' '}
+                          <Typography variant="body1">{layerTitle}</Typography>
+                        </div>
+                      );
+                    })}
+
+                    {tables.map(table => (
+                      <Button
+                        id={table.title}
+                        key={table.title}
+                        onClick={() => showTableClicked(table)}
+                      >
+                        <Typography variant="body1">{table.title}</Typography>
+                      </Button>
+                    ))}
+                  </Grid>
+                ),
+              )}
+            </Grid>
+          </AccordionDetails>
+        </Accordion>
+      ))}
+    </>
+  );
+}
+
+const styles = (theme: Theme) =>
+  createStyles({
+    icon: {
+      width: 18,
+      marginRight: 6,
+    },
+
+    categoryContainer: {
+      marginBottom: 16,
+      '&:last-child': {
+        marginBottom: 0,
+      },
+    },
+
+    categoryTitle: {
+      fontWeight: 'bold',
+      textAlign: 'left',
+    },
+
+    layersContainer: {
+      display: 'flex',
+      marginBottom: 8,
+    },
+
+    accordion: {
+      boxShadow: 'none',
+      '&.Mui-expanded': {
+        margin: 0,
+        '&:before': {
+          opacity: 1,
+        },
+      },
+    },
+
+    accordionSummary: {
+      backgroundColor: theme.palette.primary.main,
+      minHeight: 56,
+      '&.Mui-expanded': {
```

Comment:
Moving this to the MUI theme would be nice too :+1: let me know if it isn't feasible.

Reply:
I moved all the class related to accordion to the MUI theme 🙏